.gitignore

```diff
@@ -1 +1,3 @@
 build
+notes
```

Comment:
What creates the `notes` file? I tend to have the code short, if it is unlikely the `notes` will be created on other machines, I prefer to put such things info .git/info/exclude instead